modules/nw-ingress-integrating-route-secret-certificate.adoc

```diff
@@ -0,0 +1,62 @@
+//
+// * ingress/routes.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-ingress-integrating-route-secret-certificate_{context}"]
+= Integrating Third-Party Certificate Management
+
+The following workflow describes how to integrate third-party certificate management solutions, such as {cert-manager}, using the certificate reference field outlined under API Extensions.
+
+A new `.spec.tls.externalCertificate` field has been added to the Route's `.spec.tls` configuration. This field can be used to specify the name of a secret containing the certificate data, replacing the need to use `.spec.tls.certificate` and .`spec.tls.key`.
+
+.Prerequisites
+
+* You have access to the OpenShift Container Platform cluster as a user with the cluster-admin role.
+
+* You have installed the {cert-manager} for {product-title} version 1.12.0 or later.
+
+* You must have generated the serving certificate using third-party systems such as {cert-manager}.
+
+* Also for cert-manager, the Certificate Custom Resource (CR) must be created in the same namespace where the Route will be created.
```

Comment:
cert-manager is not a Prerequisites for this feature. This feature can work without cert-manager also. Users may choose to generate the certificate as per their feasibility

Reply:
Understood.